src/main/java/com/Employes/Employedetail/controller/postRequest.java

```diff
@@ -0,0 +1,42 @@
+package com.Employes.Employedetail.controller;
+//CLASS FOR POST REQUEST WHICH WOULD BE PASSED BY CLIENT
+public class postRequest {
```

Comment:
Bad naming convention, It could have been AddEmployeeEntity or AddEmployeeModel.
If it is a model class, it should not be in controller package.